src/main/java/com/entropyteam/entropay/employees/projects/models/ProjectType.java

```diff
@@ -0,0 +1,39 @@
+package com.entropy.entropay.employees.projects.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Entity
+@Table(name = "project_types")
+@Data
+public class ProjectType {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(nullable = false, unique = true)
+    private String name;
+
+    @Column(nullable = false)
+    private boolean enabled;
+
+    public long getId() {
```

Comment:
@Data should provide this getters.
For boolean try to use is instead of get

Reply:
Disregarded due to invalid PR. The correct one is [here.](https://github.com/entropy-code/entropay-employees/pull/6)